package.json

```
@@ -7,7 +7,8 @@
   "exports": {
     ".": {
       "require": "./lib/fast-check.js",
-      "default": "./lib/esm/fast-check.js"
+      "default": "./lib/esm/fast-check.js",
+      "import": "./lib/esm/fast-check.js"
```

```suggestion
      "import": "./lib/esm/fast-check.js",
      "default": "./lib/esm/fast-check.js"
```

It seems that Webpack does not like when default is not at the end.